apps/web/src/routes/MapExplorer/MapExplorer.js

```diff
@@ -288,42 +288,6 @@ function MapExplorer (props) {
   const [createCreatePopupVisible, setCreatePopupVisible] = useState(false)
   const [createPopupPosition, setCreatePopupPosition] = useState({ top: 0, left: 0, lat: 0, lng: 0 })
 
-  const [drawerWidth, setDrawerWidth] = useState(300) // minimum width of drawer
```

Comment:
Oh, did this logic get moved somewhere else? Or are we handling the drawer in a different way now? Whats it look like on mobile web?

Reply:
oh shoot, yes i removed that and replaced with CSS only but on a different branch. should still be here on this branch

Reply:
i tested it on mobile web